### PR TITLE
Removed C from galgebra

### PR DESCRIPTION
**Pull request by toolforger**

### CODE REVIEW

File: sympy/galgebra/printing.py [hunk 1]
```diff
@@ -30,10 +30,14 @@
 
 import os
 import sys
+
```
Comment (aktech) [inline sympy/galgebra/printing.py]: BTW, why this new line?

Comment (toolforger) [inline sympy/galgebra/printing.py]: There was an empty line above ```from sympy import```, and I thought "well if we're grouping imports here, then the grouping should show some form of consistency".

Comment (aktech) [inline sympy/galgebra/printing.py]: OK, I was guessing that. I am not sure if there are some PEP8 guidelines for such imports.

Comment (toolforger) [inline sympy/galgebra/printing.py]: I doubt there is a PEP8 guideline for that.
The justification does not refer to PEP8 anyway.

Comment (jcrist) [inline sympy/galgebra/printing.py]: https://www.python.org/dev/peps/pep-0008/#imports

>  Imports should be grouped in the following order:

> 1. standard library imports
2. related third party imports
3. local application/library specific imports

> You should put a blank line between each group of imports.

**Comment by jcrist** [comment]
Looks good. Merging.

File: sympy/galgebra/printing.py (hunk 1 above)
Comment (aktech) [inline sympy/galgebra/printing.py]: @jcrist 
Thanks for clarifying!

**Pull request merged**